cucumber-tsflow/src/binding-decorator.ts

```diff
@@ -158,7 +158,7 @@ function bindStepDefinition(stepBinding: StepBinding): void {
 
     return (bindingObject[
       matchingStepBindings[0].targetPropertyKey
-    ] as () => void).apply(bindingObject, arguments);
+    ] as () => void).apply(bindingObject, arguments as any);
```

Comment:
Is there any way to avoid the `any` cast here and on line 224?

Reply:
Yes, its possible with further changes.  I was trying to keep changes to a minimal, and this project is already littered with `any` casts.